### PR TITLE
add-marvin

### PR DESCRIPTION
**Pull request by raybellwaves**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/marvin/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  sha256: f96628c5e7292ef02cf4bc1f4e6692df23cd1622e4f8c7cfcb2190bda38619d6
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python 3.9
```
Comment (synapticarbors) [inline recipes/marvin/meta.yaml:19]: I think you need to use the `python_min` convention mentioned in https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python for new `noarch` packages. I tend to use the jinja variable `{% set python_min = "3.9" %}` explicitly at the top of the recipe.

**Review by synapticarbors — CHANGES_REQUESTED** [review]

**Comment by synapticarbors** [comment]
@raybellwaves -- It will take a little bit for `partialjson` to hit the conda-forge CDN, but I'll circle back in a little bit and restart the checks.

File: recipes/marvin/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    - jsonpatch >=1.33
+    - openai >=1.21.0
+    - prompt-toolkit >=3.0.33
+    - pydantic >=2.4.2
+    - pydantic-settings
+    - rich >=12
+    - tiktoken >=0.4.0
+    - typer >=0.9.0
+    - typing_extensions >=4.0.0
+    - tzdata >=2023.3
```
Comment (synapticarbors) [inline recipes/marvin/meta.yaml:39]: I think this needs to be `python-tzdata` to get the python library rather than the C-library of the same name.

**Comment by raybellwaves** [comment]
skipping windows as I got `conda_build.exceptions.CondaBuildUserError: Noarch package contains binary script: marvin.exe`

**Comment by raybellwaves** [comment]
friendly ping for another review @synapticarbors please

**Pull request merged**